.ddev/config.yaml

```diff
@@ -10,19 +10,12 @@ additional_hostnames: []
 additional_fqdns: []
 mariadb_version: "10.3"
```

Comment:
Maybe bump it up to 10.7

Reply:
Unfortunately ddev supports only 10.5, so setting this for now.